Source/Core/VideoBackends/OGL/TextureConverter.cpp

```diff
@@ -308,7 +311,7 @@ void EncodeToRamYUYV(GLuint srcTexture, const TargetRectangle& sourceRc, u8* des
   // We enable linear filtering, because the GameCube does filtering in the vertical direction when
   // yscale is enabled.
   // Otherwise we get jaggies when a game uses yscaling (most PAL games)
-  EncodeToRamUsingShader(srcTexture, destAddr, dstWidth * 2, dstHeight, dstStride, true);
```

Comment:
Why is the function EncodeToRamYUYV still here? This was the old XFB copy.

Reply:
Good catch!  More to remove!! 👍